portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -115,23 +171,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
    */
     private String getParameter(HttpServletRequest request, String name, String defaultValue) {
         String value = request.getParameter(name);
-        if (value.equals(null)) {
+        if (value == null) {
             return defaultValue;
         }
         return value;
     }
 
     /* Returns number of comments to display */
     private int getNumberOfCommentsToDisplay(HttpServletRequest request) {
-    // Get the input from the form.
-        String numberOfCommentsString = request.getParameter("comments-choice");
-    // Convert the input to an int.
+        // Get the input from the form.
+        //String numberOfCommentsString = request.getParameter("comments-choice");
```

Comment:
Remove the commented-out code. If you need to remember it again, you can use gitHub's version control to see what your older code looked like.